nhais-adaptor/adaptor/incoming/tests/test_operation_definition_adaptor.py

```diff
@@ -63,11 +75,60 @@ def test_create_operation_definition(self):
+                                                                           transaction_number="18")
+            deduction_1 = fixtures.create_operation_definition_for_deduction(recipient="TES5",
+                                                                             transaction_number="19",
+                                                                             nhs_number="NHSNO11111")
+            deduction_2 = fixtures.create_operation_definition_for_deduction(recipient="TES5",
+                                                                             transaction_number="20",
+                                                                             nhs_number="NHSNO22222")
+            expected = [("17", "TES5", approval_1), ("18", "TES5", approval_2),
+                        ("19", "TES5", deduction_1), ("20", "TES5", deduction_2)]
+
+            incoming_interchange = Interchange(InterchangeHeader("XX11", "TES5", "190429:1756"),
+                                               Messages([
+                                                   MessageSegment(MessageSegmentBeginningDetails("F4"),
+                                                                  Transactions([
+                                                                      Transaction(
+                                                                          TransactionRegistrationDetails("17")
+                                                                      ),
+                                                                      Transaction(
+                                                                          TransactionRegistrationDetails("18")
+                                                                      )
+                                                                  ])),
+                                                   MessageSegment(MessageSegmentBeginningDetails("F2"),
+                                                                  Transactions([
+                                                                      Transaction(
+                                                                          TransactionRegistrationDetails("19"),
+                                                                          TransactionPatientDetails("NHSNO11111")
+                                                                      ),
+                                                                      Transaction(
+                                                                          TransactionRegistrationDetails("20"),
+                                                                          TransactionPatientDetails("NHSNO22222")
+                                                                      )
+                                                                  ]))
+                                               ]))

```

Comment:
Bit picky but if you put the `MessageSegmentBeginningDetails` on a new line pycharm can format all the other indents along side it making the indentation a bit more readable here.

Reply:
nice 👍 that does look better.